examples/planning/generate_keyword_ideas.py

```diff
@@ -0,0 +1,148 @@
+
+    try:
+        keyword_ideas = keyword_plan_idea_service.generate_keyword_ideas(
+            customer_id, language, locations, keyword_plan_network,
+            url_seed=url_seed, keyword_seed=keyword_seed,
+            keyword_and_url_seed=keyword_url_seed)
+
+        for idea in keyword_ideas.results:
+            competition_value = keyword_competition_level_enum.Name(
+                idea.keyword_idea_metrics.competition)
+            print('Keyword idea text "%s" has %d average monthly searches and '
+                  '"%s" competition.\n' % (
+                      idea.text.value,
+                      idea.keyword_idea_metrics.avg_monthly_searches.value,
+                      competition_value))
+    except GoogleAdsException as ex:
+        print('Request with ID "%s" failed with status "%s" and includes the '
+              'following errors:' % (ex.request_id, ex.error.code().name))
+        for error in ex.failure.errors:
+            print('\tError with message "%s".' % error.message)
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print('\t\tOn field: %s' % field_path_element.field_name)
+        sys.exit(1)
+
+
+def map_keywords_to_string_values(client, keywords):
+    keyword_protos = []
+    for keyword in keywords:
+        string_val = client.get_type('StringValue')
+        string_val.value = keyword
+        keyword_protos.append(string_val)
+    return keyword_protos
+
+
+def map_locations_to_string_values(client, location_ids):
+    gtc_service = client.get_service('GeoTargetConstantService', version='v1')
+    locations = []
+    for location_id in location_ids:
+        location = client.get_type('StringValue')
+        location.value = gtc_service.geo_target_constant_path(location_id)
+        locations.append(location)
+    return locations
+
+
+def map_language_to_string_value(client, language_id):
+    language = client.get_type('StringValue')
+    language.value = client.get_service('LanguageConstantService',
+                                        version='v1').language_constant_path(
+                                            language_id)
+    return language
+
+
+if __name__ == '__main__':
+    # GoogleAdsClient will read the google-ads.yaml configuration file in the
+    # home directory if none is specified.
+    google_ads_client = GoogleAdsClient.load_from_storage()
+
+    parser = argparse.ArgumentParser(
+        description='Generates keyword ideas from a list of seed keywords.')
+
+    # The following argument(s) should be provided to run the example.
+    parser.add_argument('-c', '--customer_id', type=six.text_type,
+                        required=True, help='The Google Ads customer ID.')
+    # For more information on determining location IDs, see:
+    parser.add_argument('-k', '--keywords', type=six.text_type, required=False,
+                        help='Comma-separated starter keywords')
```

Comment:
Might as well be consistent. I think we frequently use "delimited" to describe this, and you do so below as well.